HomeWork9/src/Main.java

```diff
@@ -0,0 +1,61 @@
+public class Main {
+    public static void main(String[] args) {
+        task1();
+        task2();
+        task3();
+        task4();
+    }
+
+    public static int[] generateRandomArray() {
+        java.util.Random random = new java.util.Random();
+        int[] arr = new int[30];
+        for (int i = 0; i < arr.length; i++) {
+            arr[i] = random.nextInt(100_000) + 100_000;
+        }
+            return arr;
+    }
+    public static void task1() {
+        int[] arr = generateRandomArray();
+        int sum = 0;
+        for (int element : arr) {
+            sum = sum + element;
+        }
+        System.out.println("Сумма трат за месяц составила " + sum + " рублей");
+        System.out.println();
+    }
+
+    public static void task2() {
+        int[] arr = generateRandomArray();
+        int min = 200_000;
+        int max = -1;
+
+        for (int element : arr) {
+            if (element > max) {
+                max = element;
+            }
+            if (element < min) {
+                min = element;
+            }
+        }
+        System.out.println("Минимальная сумма трат за день составила " + min + " рублей. Максимальная сумма трат за день составила " + max + " рублей");
+        System.out.println();
+    }
+
+    public static void task3() {
+        int[] arr = generateRandomArray();
+        int sum = 0;
+        for (int element : arr) {
+            sum = sum + element;
+        }
+        float average = sum / 30;
```

Comment:
```suggestion
        float average = sum / 30F;
```
здесь отбрасывается остаток, т.е. 5 / 2 у тебя будет 2, а не 2.5, как ты ожидаешь. Чтобы было с остатком, надо привести хотя бы один из операндов к float